opm/simulators/wells/GasLiftSingleWell.hpp

```diff
@@ -69,7 +61,7 @@ namespace Opm
         void setAlqMaxRate_(const GasLiftOpt::Well& well);
 
         const Simulator &ebos_simulator_;
-        const StdWell &std_well_;
+        const WellInterface<TypeTag> &std_well_;
     };
```

Comment:
Please rename to well_  
and also:
const WellInterfaceGeneric &getStdWell() const override { return std_well_; }
to getWell()
to reflect the change from stdWell well to wellInterface. 
I

Reply:
Good idea! See added commit.